docs/alternator/compatibility.md

```diff
@@ -16,7 +16,16 @@ provision that - Scylla requires you to provision your cluster. You need
 to reason about the number and size of your nodes - not the throughput.
 
 When creating a table, the BillingMode and ProvisionedThroughput options
-are ignored by Scylla.
+are ignored by Scylla. Tables default to a Billing mode of `PAY_PER_REQUEST`
+and `DescribeTable` API calls will return a value of 0 for the RCUs, WCUs
+and NumberOfDecreasesToday within the response.
+
+Another notable difference between DynamoDB and Scylla comes down to the
```

Comment:
I think this paragraph doesn't belong in the "Provisioning" heading.
I hesitate if users need to know about it at all (after all, I don't think they have any guarantee that even DynamoDB won't change their hash function next week), but if they do, it should be in a different section. Maybe we can have a section heading "Hash-key hash function" or something like that?